### PR TITLE
Fix map_raw_ops_alias wrapping the source function without wraps decorator causing unsupported dtypes not showing

### LINKED ISSUE #20963: [Bug] Fix `unsupported_dtypes` and `supported_dtypes` for alias functions in raw_ops tensorflow frontend
Ideally, calling `ivy.function_unsupported_dtypes` on `tf.math.negative` and `tf.raw_ops.Neg` should yield the same results.
Instead this is the result at the moment:
```python
import ivy
import ivy.functional.frontends.tensorflow as tf

### def of: tf.raw_ops.Neg
# Neg = to_ivy_arrays_and_back(map_raw_ops_alias(tf_frontend.math.negative))

### def of: tf_frontend.math.negative
# @to_ivy_arrays_and_back
# def negative(x, name=None):
#     return ivy.negative(x)

### Short def of ivy.negative in Tensorflow backend:
# @with_unsupported_dtypes(
#     {"2.9.1 and below": ("uint8", "uint16", "uint32", "uint64")}, backend_version
# )
# def negative(x: Union[float, tf.Tensor, tf.Variable])...

ivy.set_backend("tensorflow")
print(ivy.function_unsupported_dtypes(tf.math.negative))
# ('uint8', 'uint32', 'uint64', 'uint16')
print(ivy.function_unsupported_dtypes(tf.raw_ops.Neg))
# ()
```

### PR DESCRIPTION
**Pull request by fabriziocacicia**
Fix #20963

### CODE REVIEW

File: ivy/functional/frontends/tensorflow/func_wrapper.py [hunk 1]
```diff
@@ -223,6 +224,8 @@ def _wraped_fn(**kwargs):
             return fn(**kwargs)
 
         _wraped_fn.__signature__ = new_signature
+        _wraped_fn.__module__ = new_module
+        _wraped_fn.__alias__ = {fn.__name__: fn}
```
Comment (CatB1t) [inline ivy/functional/frontends/tensorflow/func_wrapper.py:228]: Does it have to be stored as a dictionary?

File: ivy/functional/frontends/tensorflow/func_wrapper.py [hunk 2]
```diff
@@ -215,6 +215,7 @@ def _wrap_raw_ops_alias(fn: callable, kw_update: Dict) -> callable:
             )
             new_params.append(param.replace(name=name, kind=param.KEYWORD_ONLY))
         new_signature = sig.replace(parameters=new_params)
+        new_module = inspect.stack()[2][0].f_globals["__name__"]
```
Comment (CatB1t) [inline ivy/functional/frontends/tensorflow/func_wrapper.py:218]: What does this line here exactly do? it seems unclear, is it retrieving the current module?

File: ivy/functional/ivy/data_type.py [hunk 3]
```diff
@@ -76,6 +76,8 @@ def _lstrip_lines(source: str) -> str:
 
 # Get the list of function used the function
 def _get_function_list(func):
+    if hasattr(func, "__alias__"):
```
Comment (CatB1t) [inline ivy/functional/ivy/data_type.py:79]: Is this addition required?

**Review by CatB1t — CHANGES_REQUESTED** [review]

**Comment by Daniel4078** [comment]
This PR is closed due to inactivity for too long.

**Pull request closed without merging**